homeassistant/components/wled/update.py

```diff
@@ -96,7 +96,9 @@ def release_url(self) -> str | None:
         """URL to the full release notes of the latest version available."""
         if (version := self.latest_version) is None:
             return None
-        return f"https://github.com/Aircoookie/WLED/releases/tag/v{version}"
+        if(re.search("^[0-9]", version)):
```

Comment:
Regular expressions might be overkill here.
```suggestion
        if version[:1].isdigit():
```

Reply:
I know virtually no python, your solution is nearer. Initially I was going to do full semver checking regex then decided that was overkill.
Especially as I'm the current release manager for WLED so I know what the two styles I'll be using is